tests/test_subber.py

```diff
@@ -8,18 +9,17 @@ class TestSubber(flask_testing.TestCase):
 
     @patch('subber.reddit.Reddit')
     def create_app(self, mock_session):
-        # Mock API session on module import
```

Comment:
The import statement will actually have to occur here due to the limitations in the design of the Reddit class (an issue will be added to address this).

If you are still able/willing to make changes to this PR, can you mock the session return value along with the Redditor before the import (instead of doing so in every unit test)? In order to accomplish this, the import statement will actually have to be on line 13.

Reply:
I can do that; no problem.